packages/app/src/app/components/CreateNewSandbox/CreateSandbox/useTeamTemplates.ts

```diff
@@ -18,17 +26,15 @@ function getTeamTemplates(data: ListPersonalTemplatesQuery, teamId: string) {
   return data.me.teams.find(team => team.id === teamId).templates;
```

Comment:
For some reason, on `/s` the `find` method returns `undefined` leading the app to crash when trying to access the `templates` property. Tested using the codesandbox team in production.

<img width="800" alt="Screen Shot 2022-09-28 at 19 33 04" src="https://user-images.githubusercontent.com/24959348/192900295-b90df9a1-db98-4f34-b65a-1fce41837083.png">

Reply:
weird, what teams do you have there? maybe a data issue? i'm adding a check if the team is not found to return []